presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -755,6 +755,42 @@ Procedures
     Unregisters given, existing partition in the metastore for the specified table.
     The partition data is not deleted.
 
+Special Columns
+---------------
+
+In addition to the defined columns, the Hive connector automatically exposes
+metadata in a number of hidden columns in each table. You can use these columns
+in your SQL statements like any other column, e.g. to return the data or to use
+them in condition statements.
+
+* ``$bucket``
+    Name of the bucket.
+
+* ``file_modified_time``
+    Date and time of the last modification of the file in the storage.
+
+* ``file_size``
+    Size of the file in the storage.
+
+* ``$path``
+    Full path to the file in the storage.
+
+* ``$shard_row_id``
+    Identifier of the shard row used for metadata deletion in Presto.
+
+Table Properties
+----------------
+
+Presto maintains a separate table that contains the table properties such as
```

Comment:
These are actually Hive table properties, not Hive connector properties, so none of the names here are valid. I don't think any of the property names are actually useful to list here, as they are only something you'd look at if you already know what you are looking for. (it's mostly for debugging)
```
The raw Hive table properties are available as a hidden table, containing a separate
column per table property, with a single row containing the property values. The
properties table name is the same as the table name with ``$properties`` appended.

You can inspect the property names and values with a simple query:

    SELECT * FROM hive.web."page_views$properties";

Reply:
oh .. makes sense. I will fix tomorrow